Services/CustomDeviceModels.cs

```diff
@@ -67,7 +71,9 @@ public async Task<IEnumerable<DeviceModel>> GetListAsync()
             }
             catch (Exception e)
             {
-                this.log.Error("Unable to load device models from storage", e);
+                string msg = "Unable to load device models from storage";
```

Comment:
Why are you creating separate variable for this message when this is used only once? Same applies to other log locations..